domain/src/main/kotlin/no/nav/su/se/bakover/domain/grunnlag/Grunnlagsdata.kt

```diff
@@ -76,6 +82,41 @@ data class Grunnlagsdata private constructor(
                 )
             }
         }
+
+        /**
+         * Tillater at vi oppretter med ufullstendig bosituasjon for å støtte søknadsbehandlinger i tidlige faser,
+         * og/eller avslag før sats er tatt stilling til.
+         */
+        fun tryCreateTillatUfullstendigBosituasjon(
+            fradragsgrunnlag: List<Fradragsgrunnlag>,
+            bosituasjon: List<Bosituasjon>,
+        ): Either<KunneIkkeLageGrunnlagsdata, Grunnlagsdata> {
+            return SjekkOmGrunnlagErKonsistent.BosituasjonOgFradrag(
+                bosituasjon = bosituasjon,
+                fradrag = fradragsgrunnlag,
+            ).resultat.mapLeft { problemer ->
+                when (val feil = problemer.first()) {
+                    is Konsistensproblem.BosituasjonOgFradrag.UgyldigBosituasjon -> {
+                        if (feil.feil.filterNot { it is Konsistensproblem.Bosituasjon.Ufullstendig }.isNotEmpty()) {
+                            KunneIkkeLageGrunnlagsdata.Konsistenssjekk(feil)
+                        } else {
+                            return Grunnlagsdata(
+                                fradragsgrunnlag = fradragsgrunnlag.sortedBy { it.periode },
+                                bosituasjon = bosituasjon.sortedBy { it.periode },
+                            ).right()
+                        }
+                    }
+                    else -> {
+                        KunneIkkeLageGrunnlagsdata.Konsistenssjekk(feil)
+                    }
+                }
+            }.map {
```

Comment:
Er ikke dette allerede gjort på linje 103? d.v.s att dette kan fjernes

Reply:
Nei, dette må fortsatt være der for alle caser hvor det ikke eksisterer noen problmer.